<launch>
  <node type="rviz" name="rviz" pkg="rviz" args="-d /home/alberto/workspaces/workspace14diciembre/TFM.rviz" />
  <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="1.09  0.237 0  1.5707963268 0 0  base_link velodyne_i 100" />
  <node pkg="tf" type="static_transform_publisher" name="link2_broadcaster" args="1.09 -0.237 0 -1.5707963268 0 0  base_link velodyne_d 100" />
  <rosparam command="load" file="params.yaml" />
  <param name="test" value="2" />
  <param name="tfEKF" value="$(arg tfEKF)" />
  <param name="logfile" value="$(arg logfile)" />
  <node pkg="pcl_segment" type="odom2" name="odom2" />
  <node pkg="pcl_segment" type="GPSLogger" name="GPSLogger" />
  <node pkg="pcl_segment" type="pointcloud_merge" name="pointcloud_merge"/>
  <node pkg="pcl_segment" type="euclidean_clustering" name="euclidean_clustering"/>
  <node pkg="pcl_segment" type="ekf" name="ekf" output="screen"/>
  <node pkg="rosbag" type="play" name="player" args="--clock /home/alberto/workspaces/workspace14diciembre/Validacion/prueba2_select.bag"/>
</launch>
